rudder-web/src/main/webapp/javascript/rudder/rudder-datatable.js

```diff
@@ -486,7 +486,7 @@ function createRuleTable(gridId, data, checkboxColumn, actionsColumn, compliance
       }
     , "columnDefs": [{
           "targets": [ 0 ]
-        , "visible": false
+        , "visible": true
```

Comment:
Do you really want to change that line ?

Reply:
nop :)